### DISCUSSION TITLE
Does FINN support multiheaded/residual networks?

### DISCUSSION THREAD

**Discussion by JPPalacios** [Xilinx/finn#991] [Q&A] [answered]
Hi,

I would like to know if the FINN compiler accepts shortcuts/residuals or having multiple heads for classification/localization within networks? Was interested in looking into MobileV2 architectures for eventual PYNQ deployment. 

If not, what would be the best way to "streamline" both tasks into a single output?

Thanks!

**Comment by MFJI** [comment, marked as answer]
Finn-examples has something similar: https://github.com/Xilinx/finn-examples/tree/main/build/mobilenet-v1 and https://github.com/Xilinx/finn-examples/tree/main/build/resnet50.
